src/emcpy/io/netCDF.py

```diff
@@ -79,3 +92,49 @@ def read_netCDF_var(filename, variable_name, oneD=False, ftime=-1, flevel=-1):
 
     result = _np.squeeze(var)
     return result
+
+def filter_obs(code,codes,errorinv,lat,lon,pressure,\
```

Comment:
This should probably be somewhere else but not immediately sure where. Any suggestions @aerorahul or @kevindougherty-noaa ?

Reply:
I made a utils directory and move this function there.